server/routes/users-routes.js

```diff
@@ -38,7 +38,7 @@ router.get(
 
 router.get('/topcategories/:userid', usersController.getTopCategories);
 
-router.get('/monthlytransactions/:userid&:month&:timezone', 
+router.get('/monthlytransactions/:userid&:year&:month&:timezone', 
```

Comment:
I would rename this router to
```suggestion
router.get('/monthlyTransaction/:userid&:year&:month&:timezone', 
```